servant-server/src/Servant/Server/Internal/RouterEnv.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE DeriveFunctor     #-}
```

Comment:
Could you add

* [ ] Module-level haddocks
* [ ] `@since` annotations on these elements?

Reply:
And there are extraneous empty lines between declarations